aruna/api/storage/services/v1/collection_service.proto

```diff
@@ -303,4 +316,15 @@ message DeleteCollectionRequest {
   bool force = 3;
 }
 
-message DeleteCollectionResponse {}
\ No newline at end of file
+message DeleteCollectionResponse {}
+
+
+message AddKeyValueToCollectionRequest {
```

Comment:
Same as before, this should be changed to `KeyValues` to convey that more than one Label/Hook can be added.

Reply:
Fixed by [24fc061](https://github.com/ArunaStorage/ArunaAPI/pull/79/commits/24fc061dd1efcc18cf4e1bd68ca04c792e52b35d)